packages/base/any/kernels/3.16-lts/patches/driver-i2c-bus-intel-ismt-netberg-aurora.patch

```diff
@@ -0,0 +1,16 @@
+diff -Nu a/drivers/i2c/busses/i2c-ismt.c b/drivers/i2c/busses/i2c-ismt.c
```

Comment:
We found that some AOC locked the i2c-ismt bus at the beginning of inserting in machine and it causes i2c bus access timeout when reading EEPROM data from it, so we added "retry" here.